pytket/extensions/qiskit/qiskit_convert.py

```diff
@@ -653,22 +666,31 @@ def append_tk_command_to_qiskit(
             width = op.width  # type: ignore
             value = op.value  # type: ignore
         regname = args[0].reg_name
-        if len(cregmap[regname]) != width:
-            raise NotImplementedError("OpenQASM conditions must be an entire register")
         for i, a in enumerate(args[:width]):
             if a.reg_name != regname:
                 raise NotImplementedError(
                     "OpenQASM conditions can only use a single register"
                 )
-            if a.index != [i]:
-                raise NotImplementedError(
-                    "OpenQASM conditions must be an entire register in order"
-                )
         instruction = append_tk_command_to_qiskit(
             op.op, args[width:], qcirc, qregmap, cregmap, symb_map, range_preds  # type: ignore
         )
+        if len(cregmap[regname]) == width:
+            for i, a in enumerate(args[:width]):
+                if a.index != [i]:
+                    raise NotImplementedError(
+                        """OpenQASM conditions must be an entire register in
+ order or only one bit of one register"""
+                    )
+
+            instruction.c_if(cregmap[regname], value)
+        elif width == 1:
+            instruction.c_if(cregmap[regname][args[0].index[0]], value)
+        else:
+            raise NotImplementedError(
+                """OpenQASM conditions must be an entire register in
+order or only one bit of one register"""
```

Comment:
Can we make this a string with no newlines?

Reply:
done in @cqc-melf
[remove linebreak from error message](https://github.com/CQCL/pytket-qiskit/pull/245/commits/f2a4770491ea359b7c93e021be5d46220611e92a)
[f2a4770](https://github.com/CQCL/pytket-qiskit/pull/245/commits/f2a4770491ea359b7c93e021be5d46220611e92a)